app/assets/javascripts/admin/variant_overrides/controllers/variant_overrides_controller.js.coffee

```diff
@@ -52,55 +52,55 @@ angular.module("admin.variantOverrides").controller "AdminVariantOverridesCtrl",
 
   $scope.displayDirty = ->
     if DirtyVariantOverrides.count() > 0
-      num = if DirtyVariantOverrides.count() == 1 then "one override" else "#{DirtyVariantOverrides.count()} overrides"
-      StatusMessage.display 'notice', "Changes to #{num} remain unsaved."
+      num = if DirtyVariantOverrides.count() == 1 then t('js.variant_overrides.one_override')  else "#{DirtyVariantOverrides.count()} " + t('js.variant_overrides.overrides')
+      StatusMessage.display 'notice', t('js.variant_overrides.changes_to') + ' ' + num + ' ' + t('js.variant_overrides.remain_unsaved')
     else
       StatusMessage.clear()
 
   $scope.update = ->
     if DirtyVariantOverrides.count() == 0
-      StatusMessage.display 'alert', 'No changes to save.'
+      StatusMessage.display 'alert', t('js.variant_overrides.no_changes_to_save')
     else
-      StatusMessage.display 'progress', 'Saving...'
+      StatusMessage.display 'progress', t('js.saving')
       DirtyVariantOverrides.save()
       .success (updatedVos) ->
         DirtyVariantOverrides.clear()
         VariantOverrides.updateIds updatedVos
         $scope.variant_overrides_form.$setPristine()
-        StatusMessage.display 'success', 'Changes saved.'
+        StatusMessage.display 'success', t('js.changes_saved')
         VariantOverrides.updateData updatedVos # Refresh page data
       .error (data, status) ->
         StatusMessage.display 'failure', $scope.updateError(data, status)
 
 
   $scope.updateError = (data, status) ->
     if status == 401
-      "I couldn't get authorisation to save those changes, so they remain unsaved."
+      t('js.variant_overrides.no_authorisation')
 
     else if status == 400 && data.errors?
       errors = []
       for field, field_errors of data.errors
         errors = errors.concat field_errors
       errors = errors.join ', '
-      "I had some trouble saving: #{errors}"
+      t('js.variant_overrides.some_trouble') + ": #{errors}"
```

Comment:
@jeronimo, isn't that the point of interpolation? You can choose whether use the colon or not in the translation. If we keep it the way it is, the colon is hard-wired, and so it will be impossible to come up with a translation that makes sense in other languages. See @mkllnk's comment above. This would allow you to include a colon in the english translation, but structure it completely differently in another language.